curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60fab4a123ce4b04526b082b.md

```diff
@@ -9,7 +9,7 @@ dashedName: step-28
 
 Currently when someone submit the form, they can submit it without checking the radio inputs. Although you had used `required` attribute to indicate the the input is required previously, this can't work in this case, because adding required to both inputs, will convey the wrong information to the form users.
 
-To solve this, you can provide context of what is needed by adding `legend` element below the second `fieldset` with text `Account type (required)`, then add `checked` attribute to the `Personal` input to make sure that the form is submitted with the required data in it.
+To solve this, you can provide context of what is needed by adding a `legend` element below the second `label` with text `Account type (required)`, then add `checked` attribute to the `Personal` input to make sure that the form is submitted with the required data in it.
```

Comment:
```suggestion
To solve this, you can provide context of what is needed by adding a `legend` element with text `Account type (required)` before the `label` elements within the second `fieldset`. Then add the `checked` attribute to the `Personal` input to ensure the form is submitted with the required data in it.
```

You don't have to use this exact wording, but the main change is that the `legend` should be place above/before the `label` elements and inside of the second `fieldset`.

Reply:
Should I open an Issue for this first? I was treating it as a typo